composeApp/src/commonMain/sqldelight/org/ooni/probe/data/TestDescriptor.sq

```diff
@@ -71,7 +71,7 @@ setAutoUpdate:
 UPDATE TestDescriptor SET auto_update = ? WHERE runId = ?;
 
 selectAll:
-SELECT * FROM TestDescriptor;
+SELECT * FROM TestDescriptor WHERE (is_expired IS NULL OR is_expired = :isExpired) OR (:isExpired IS NULL);
```

Comment:
Although we have a `is_expired` field inside the DB, we shouldn't. It's not a reliable way to know if the descriptor is actually expired, because it won't be updated every day. We only update it correctly if the user as enabled auto-update. Otherwise, it may be already expired, but the DB field is not updated.

Either we update the `is_expired` field for all descriptors consistently, or we should drop it from the DB and just use the expired date.